### PR TITLE
Core/ts version task 427 event name structure

### PR DESCRIPTION
**Pull request by franken5tein**
## Check list  

- [x] All Merge Request pipelines should be passed.
- [x] Make sure all the changes are necessary and correct.
- [x] Make sure no needed change missing (all necessary changes are made).
- [x] Merge Request should contains only changes for one purpose (one feature or bug or refactoring).
- [ ] All necessary unit tests are written.
- [x] Critical changes should be tested locally (or on staging).

### CODE REVIEW

File: ts-version/src/data/database/postgres/repository/fill.adapter.ts [hunk 1]
```diff
@@ -172,7 +173,7 @@ class FillRepoAdapter extends TransactionalDB implements IFillRepo.schema {
           dataValues.created_at,
           dataValues.tx_hash,
         );
-        Fill.notify("fill_updated", fill);
+        Fill.notify(FillEventName.Updated, fill);
```
Comment (GreatGaryCooper) [inline ts-version/src/data/database/postgres/repository/fill.adapter.ts:176]: Just a comment, we can omit the `Name` part and let it just be `FillEvent.Updated`.

File: ts-version/src/data/price-api/ccxt.ts [hunk 2]
```diff
@@ -67,7 +68,7 @@ export class PriceAPI extends Observable implements IPriceFetcher {
       const marketStats = await this.fetchStats(market);
       if (marketStats) {
         this._marketsStats.set(market, marketStats);
-        PriceAPI.notify("stats_updated", marketStats);
+        PriceAPI.notify(StatusEventName.Updated, marketStats);
```
Comment (GreatGaryCooper) [inline ts-version/src/data/price-api/ccxt.ts:71]: It should be `StatsEvent.Updated`, not `StatusEven.Updated`

File: ts-version/src/domain/entities/fill.entity.ts [hunk 3]
```diff
@@ -45,7 +46,7 @@ export default class Fill extends Observable {
       createdAt,
     );
 
-    Fill.notify("fill_creating", fill);
+    Fill.notify(FillEventName.Created, fill);
```
Comment (GreatGaryCooper) [inline ts-version/src/domain/entities/fill.entity.ts:49]: This field used to be `fill_creating`, but it is `fill_created`, is there a reason for it?
Have we changed the logic?

File: ts-version/src/domain/entities/order/order.entity.ts [hunk 4]
```diff
@@ -235,7 +236,7 @@ export default class Order extends Observable {
 
     await order.runMakePolicies();
 
-    Order.notify("order_creating", order);
+    Order.notify(OrderEventName.Updating, order);
```
Comment (GreatGaryCooper) [inline ts-version/src/domain/entities/order/order.entity.ts:239]: This used to be `order_creating`, but now it is `order_updating`, is it on purpose?

File: ts-version/src/domain/types/event.type.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+enum FillEventName {
+  Created = "fill_created",
+  Creating = "fill_creating",
+  Updated = "fill_updated",
+  Updating = "fill_updating",
+}
+
+enum StatusEventName {
```
Comment (GreatGaryCooper) [inline ts-version/src/domain/types/event.type.ts:15]: As per the comment above, this should be `Stats`, not `Status`

**Review by GreatGaryCooper — CHANGES_REQUESTED** [review]

File: ts-version/src/data/database/postgres/repository/fill.adapter.ts (hunk 1 above)
Comment (franken5tein) [reply ts-version/src/data/database/postgres/repository/fill.adapter.ts]: Done!

File: ts-version/src/data/price-api/ccxt.ts (hunk 2 above)
Comment (franken5tein) [reply ts-version/src/data/price-api/ccxt.ts]: Done!

File: ts-version/src/domain/types/event.type.ts (hunk 5 above)
Comment (franken5tein) [reply ts-version/src/domain/types/event.type.ts]: Done!

**Review by GreatGaryCooper — APPROVED** [review]
LGTM

File: ts-version/src/presentation/http/middlewares/authentication.ts [hunk 6]
```diff
@@ -5,29 +5,30 @@ import {
   UnauthorizedError,
   UnauthorizedErrorReason,
 } from "@error/unauthorized.error";
+import { IUserInfo } from "@type/user-info.type";
+
+declare global {
```
Comment (hypersolo) [inline ts-version/src/presentation/http/middlewares/authentication.ts:10]: move this to typing directory

**Comment by hypersolo** [comment]
jwt-payload-validator middleware is not added in GET user/fills endpoint

**Pull request merged**